pkg/hvutils/hvutils_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hvutils
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getInstanceTypeFromTags(t *testing.T) {
+	tests := []struct {
+		name string
+		tags []string
+		want string
+		err  error
+	}{
+		{
+			name: "empty slice returns empty string",
+			tags: []string{},
+			want: "",
+			err:  errors.New(errors.New("no instance-type tag found on deviceID=1").Error()),
```

Comment:
This is redundant, isn't it? One errors.New should be enough

Reply:
can you in general use fmt.Errorf? We have used that in the other cases.

Reply:
Yes, you are right. fmt.Errorf() works fine.